src/commands/queue.ts

```diff
@@ -37,20 +53,17 @@ export function addQueueCommands(bot: DiscordBot, config: BotConfig, daos: Daos)
     },
 
     'add': async (message: Message, params: string[]) => {
-      try {
-        if (params.length === 0) return await message.reply("You didn't specify what to add!");
-        const queryResults = await Utils.songQuery(message, params.join(' '), scUsers, users, scApi, ytApi);
-        if (_.isNil(queryResults)) return;
-        await queuePlayerManager.get(message.guild.id).enqueue(queryResults.songs, queryResults.nextFlag);
-        const nameOrLength = queryResults.songs.length > 1 ? `${queryResults.songs.length} songs` : `**${queryResults.songs[0].title}**`;
-        let addedMsg = `Successfully added ${nameOrLength} `;
-        addedMsg += queryResults.nextFlag ? 'to be played next!' : 'to the queue!';
-        await message.reply(addedMsg);
-      } catch (e) {
-        await message.reply(`Failed to add anything to the queue.`);
-        console.log(e);
+      add(message, params)
+    },
+
+    'replace': async (message: Message, params: string[]) => {
+      const queuePlayer = queuePlayerManager.get(message.guild.id);
+      if (queuePlayer.queuedTracks !== 0) {
+        await queuePlayer.clear();
+        await message.reply('I have cleared the queue');
```

Comment:
See comment above - if we go back to duplicate logic you can update the replies to say `"Successfully replaced queue with `${nameOrLength}"` or something. And probably no need to reply that we cleared the queue but rather replaced it :)

Reply:
Now it replies: `"Successfully replaced queue with ${PLAYLISTS_NAMES} (${SONG_COUNT} songs)!"` or `"Successfully replaced queue with ${SONG_NAME}!"` if only one song is replaced.